src/Catel.Test/Catel.Test.Shared/Collections/FastObservableCollectionFacts.cs

```diff
@@ -138,37 +137,37 @@ public void RaisesSingleAddEventWhileAddingRangeInSuspensionModeAdding()
                 fastCollection.AutomaticallyDispatchChangeNotifications = false;
                 fastCollection.CollectionChanged += (sender, e) => eventArgs = e;
 
-                fastCollection.AddItems(new[] { 1, 2, 3, 4, 5 }, SuspensionMode.Adding);
+                fastCollection.AddItems(new[] {1, 2, 3, 4, 5}, SuspensionMode.Adding);
 
                 Assert.AreEqual(NotifyCollectionChangedAction.Add, eventArgs.Action);
             }
 
             [TestCase]
-            public void RaisesSingleResetEventWhileAddingRangeInSuspensionModeNone()
+            public void RaisesSingleAddEventWhileAddingRangeInSuspensionModeNone()
             {
                 var eventArgs = default(NotifyCollectionChangedEventArgs);
 
                 var fastCollection = new FastObservableCollection<int>();
                 fastCollection.AutomaticallyDispatchChangeNotifications = false;
                 fastCollection.CollectionChanged += (sender, e) => eventArgs = e;
 
-                fastCollection.AddItems(new[] { 1, 2, 3, 4, 5 }, SuspensionMode.None);
+                fastCollection.AddItems(new[] {1, 2, 3, 4, 5}, SuspensionMode.None);
 
-                Assert.AreEqual(NotifyCollectionChangedAction.Reset, eventArgs.Action);
+                Assert.AreEqual(NotifyCollectionChangedAction.Add, eventArgs.Action);
```

Comment:
This test doesn't look right. We need:

1. A counter to actually count
2. Assert the count

While reading this code, I think we will have 5 Add events (since suspensionmode == null). Then either the test or the title is wrong.